### ISSUE TITLE
Ruby 3.0.0 tried to create Proc object without a block (ArgumentError)

### ISSUE THREAD

**Issue by durcak** [anycable/anycable#118]
### Tell us about your environment

**Ruby version:** 3.0.0

**`anycable` gem version:** anycable (1.0.2)

**`grpc` gem version:** grpc (1.36.0)

### What did you do?
Upgrade ruby version from 2.7.2 to 3.0.0

### What did you expect to happen?
Everything work as before :)

### What actually happened?
Anycable failed with error: `tried to create Proc object without a block`

Backtrace:
```
{"host":"backend-6674b567f7-5cwqj","application":"Semantic Logger","environment":"production","timestamp":"2021-03-05T09:40:07.738947Z","level":"info","level_index":2,"pid":6,"thread":"6100","name":"ActionCable","message":"ruby 3.0.0 (2020-12-25) [x86_64-linux]","dd":{}}
tried to create Proc object without a block
/app/vendor/ruby/3.0.0/gems/anycable-1.0.2/lib/anycable/cli.rb:108:in `new'
/app/vendor/ruby/3.0.0/gems/anycable-1.0.2/lib/anycable/cli.rb:108:in `at_stop'
/app/vendor/ruby/3.0.0/gems/anycable-1.0.2/lib/anycable/cli.rb:189:in `start_health_server!'
/app/vendor/ruby/3.0.0/gems/anycable-1.0.2/lib/anycable/cli.rb:67:in `run'
/app/vendor/ruby/3.0.0/gems/anycable-1.0.2/bin/anycable:7:in `<top (required)>'
/app/vendor/ruby/3.0.0/bin/anycable:23:in `load'
/app/vendor/ruby/3.0.0/bin/anycable:23:in `<top (required)>'
/usr/local/bundle/gems/bundler-2.2.8/lib/bundler/cli/exec.rb:63:in `load'
...
```

**Comment by palkan** [comment]
Thanks for reporting!

Released `anycable` 1.0.3 and `anycable-rails` 1.0.7.

**Comment by rserling** [comment]
Sorry to resurrect this issue, but I'm not seeing the PR or Release associated with this fix. Could someone point me to the diff of what was changed to fix this problem?
